test/python/common/nvme.py

```diff
@@ -121,7 +121,7 @@ def nvme_find_ctrl(uri):
 
     # Finds correct Device
     devs = list(filter(lambda d: nqn in d.get("SubsystemNQN"), discover.get("Devices")))
-    assert len(devs) is 1, "Multiple devices with the same subnqn"
+    assert len(devs) == 1, "Multiple devices with the same subnqn"
```

Comment:
What's the difference? `is` should also work right?

Reply:
it's a warning with pytest